pkg/compute/manifest/manifest.go

```diff
@@ -183,17 +183,26 @@ func (v *Version) UnmarshalText(text []byte) error {
 // File represents all of the configuration parameters in the fastly.toml
 // manifest file schema.
 type File struct {
-	ManifestVersion Version  `toml:"manifest_version"`
-	Name            string   `toml:"name"`
-	Description     string   `toml:"description"`
-	Authors         []string `toml:"authors"`
-	Language        string   `toml:"language"`
-	ServiceID       string   `toml:"service_id"`
+	ManifestVersion Version     `toml:"manifest_version"`
+	Name            string      `toml:"name"`
+	Description     string      `toml:"description"`
+	Authors         []string    `toml:"authors"`
+	Language        string      `toml:"language"`
+	ServiceID       string      `toml:"service_id"`
+	LocalServer     LocalServer `toml:"local_server"`
```

Comment:
5. We update the manifest `File` data structure to reflect there is now a new `[local_server]` section (as per the [`fastly.toml`](https://developer.fastly.com/reference/fastly-toml/) specification).

> **NOTE**: This doesn't reflect the current state of the world (the section is defined as `[testing]` in the design document). @triblondon and I have been pushing to rename 'testing', and after a discussion we both agreed on 'local_server' but we still need to get agreement from the wider development team.

Reply:
/cc @acfoltzer @cratelyn

Reply:
UPDATE: the change from `testing` to `local_server` is now agreed.